lib/s390base.pm

```diff
@@ -37,13 +49,29 @@ sub copy_testsuite {
     save_screenshot;
 }
 
+=head2 execute_script
+
+ execute_script();
+
+Execute script and upload script logs
+
+=cut
 sub execute_script {
     my ($self, $script, $scriptargs, $timeout) = @_;
     assert_script_run("./$script $scriptargs  >> $script.log 2>&1", timeout => $timeout);
     save_screenshot;
     upload_logs "$script.log";
 }
 
+=head2 cleanup_testsuite
+
+ cleanup_testsuite();
+
+Currently it just returns value 1. 
+
+It should be able to remove files saved at ./tmp
+
+=cut
 sub cleanup_testsuite {
     return 1;
     # FIXME assert_script_run 'cd / && rm -rf ./tmp';
```

Comment:
:man_facepalming: how could this end up here?

Reply:
Actually this just returns 1. 
assert_script_run 'cd / && rm -rf ./tmp'; is not working I'm afraid. see #FIXME

Reply:
okay, just see your PR :)